### PR TITLE
Dzhus/provider update

### PR DESCRIPTION
**Pull request by VyacheslavDzhus**

### CODE REVIEW

**Review by Elizabeth129 — APPROVED** [review]

**Comment by dmitrykiev** [comment]
please address this code smell
https://sonarcloud.io/project/issues?id=ita-social-projects-oos-backend&open=AXj-f4o01nk9lrkwtCEV&pullRequest=110&resolved=false&types=CODE_SMELL

File: OutOfSchool/OutOfSchool.DataAccess/Models/Provider.cs [hunk 1]
```diff
@@ -9,13 +9,13 @@ public class Provider
...
-        [Required(ErrorMessage = "Title is required")]
+        [Required(ErrorMessage = "Full Title is required")]
         [DataType(DataType.Text)]
         [MaxLength(60)]
         [MinLength(1)]
-        public string Title { get; set; }
+        public string FullTitle { get; set; }
 
-        [Required(ErrorMessage = "ShortTitle is required")]
+        [Required(ErrorMessage = "Short Title is required")]
```
Comment (dmitrykiev) [inline OutOfSchool/OutOfSchool.DataAccess/Models/Provider.cs:18]: do we have localization support in the project yet?

Comment (VyacheslavDzhus) [reply OutOfSchool/OutOfSchool.DataAccess/Models/Provider.cs:18]: Yes we have, but not to all parts of the project.

File: OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs [hunk 2]
```diff
@@ -26,5 +26,29 @@ public class AddressDto
...
+            }
+            
+            var address = obj as AddressDto;
+
+            if (address == null)
+            {
+                return false;
+            }
+
+            return (Region == address.Region) && (District == address.District) && (City == address.City) && (Street == address.Street) && (BuildingNumber == address.BuildingNumber);
```
Comment (dmitrykiev) [inline OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs:44]: better use Equals for string comparison

File: OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs [hunk 3]
```diff
@@ -26,5 +26,29 @@ public class AddressDto
...
+        }
+
+#pragma warning disable CA1307 // Specify StringComparison
+        public override int GetHashCode()
+        {
+            return Region.GetHashCode() ^ District.GetHashCode() ^ City.GetHashCode() ^ Street.GetHashCode() ^ BuildingNumber.GetHashCode();
```
Comment (dmitrykiev) [inline OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs:50]: is it possible to Specify StringComparison ?

**Review by dmitrykiev — APPROVED** [review]

**Pull request merged**